ddl/ddl_worker.go

```diff
@@ -770,6 +770,10 @@ func (w *worker) HandleDDLJobTable(d *ddlCtx, job *model.Job) (int64, error) {
 	// and retry later if the job is not cancelled.
 	schemaVer, runJobErr = w.runDDLJob(d, t, job)
 
+	d.mu.RLock()
```

Comment:
Does this code only for test？

Reply:
Yes.

Reply:
Do we have a way to execute these three lines codes only in test?

Reply:
The overhead is negligible. 

Please also check the other methods of this interface. All of them are written in this style.

Reply:
I know the lock and a nil function call and unlock overhead is very small. Just want to know if there is a way like c "debug micro" that only debug mode has this code.